<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="4" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Максимум(мкм)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Размер текста на изображении</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="windowBox">
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>99999.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="minimumSizeBox">
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="selectFileButton">
      <property name="text">
       <string>Выбрать фотографию</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="maxSizeBox">
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>threshold</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSlider" name="thresholdSlider">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="textSizeBox">
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="5">
     <widget class="QLabel" name="pathLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Минимальный размер блистера(мкм)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="mkmBox">
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Коэффициент размеров mkm/pix</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QSpinBox" name="pixelBox">
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="value">
       <number>447</number>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Размер окна(мкм)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="6">
     <widget class="QCustomPlot" name="xPlot" native="true"/>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="QCustomPlot" name="yPlot" native="true"/>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>plot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
